AUDITING.md

```diff
@@ -14,37 +14,30 @@ When a chaos event is terminated through an *API*, *CLI*, or *Extension*, the fo
 
 Additionally an *Extension* may record additional information regarding the specificities of the event. This information is stored within an `extension` property.
 
-Every chaos event initiation/termination will be logged. The general structure of an event log is defined below:
+Every chaos event type will be logged. The general structure of an event log is defined below:
 
 - _event log_ (Object)
   - `id` (String) 
     - GUID unique to this event instantiation
   - `type` (String) 
     - One of `'initiate'`, `'terminate'`
+    - As extension functionality grows, more types can be added to this list. 
+    - These exist for the purpose of filtering audits.
   - `user` (String)
     - Some identifiable piece of information specifying which user executed the chaos event
   - `system` (String | Object)
     - Some identifiable piece of information specifying which system the chaos event was executed from
     - If more system information can be provided, do so in the form of an object
   - `date` (String)
     - UTC timestamp of chaos event initiation
-  - `resources` (Object)
-    - An object that specifies the resources the chaos event has been executed on
-    - `subscriptionID` (String)
-    - `_resources` (Object):
-      - _key_ attributes are Azure resource group names
-      - Each resource group property is of type (Array) and contains the list of resources names
-      - Example: 
-        ```json
-          "_resources": { 
-            "resourceGroup1": [ "resName1" , "resName2" ], 
-            ...
-          },
+  - `resource` (String)
+    - String representation of the resource targeted by the chaos
+    - This is typically generated by `chaos resgen` and looks like so: `"subId/resGrpId/resName"`
   - `event` (Object)
```

Comment:
Hmmm okay that is an interesting idea. Would we need another attribute to differentiate between the two schemas? If so what would that be? `schemaType` maybe? And then `type` could become => `eventType`.

Reply:
i'd imagine it'd be possible to differentiate given the difference of fields rather than some explicit field to check...

for example: message `a` has field `eventId` and message `b` does not, therefore `a` is of type `chaosStart`

Reply:
Alright cool. This will need to be clearly denoted in the documentation (once implemented) so developer know what to expect and the recommended way of differentiating between the two. I'll go ahead and make these changes 👍